app/src/marketplace/containers/EditProductPage/DeployDataUnionModal.jsx

```diff
@@ -72,15 +72,17 @@ export const DeployDialog = ({ product, api, updateAddress }: DeployDialogProps)
             throw new Error('no product!')
         }
 
-        const { id: joinPartStreamId } = await createJoinPartStream(productId)
+        const web3 = getWeb3()
+        const account = await web3.getDefaultAccount()
+        const { id: joinPartStreamId } = await createJoinPartStream(account, productId)
```

Comment:
This works for now, join/part stream creation is going away and will any change a bit in #1109